tests/unit/algorithms/detection/tiling/test_tiling_detection.py

```diff
@@ -67,7 +67,7 @@ def create_otx_dataset(height: int, width: int, labels: List[str]):
     """
     labels = []
     for label in ["rectangle", "ellipse", "triangle"]:
-        labels.append(LabelEntity(name=label, domain=Domain.DETECTION))
+        labels.append(LabelEntity(name=label, domain=Domain))
```

Comment:
```suggestion
        labels.append(LabelEntity(name=label, domain=domain))
```
Was this your intention?

Reply:
@goodsong81 yes! Thanks for the catch!